classify-split-extract-workflow/classify-job/docai_helper.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+from typing import Tuple, Optional, List
+from google.cloud.documentai_toolbox import document
+from google.cloud import documentai_v1 as documentai
+from config import get_parser_by_name
+from logging_handler import Logger
+
+logger = Logger.get_logger(__file__)
+
+
+def get_docai_input(processor_name: str) -> \
```

Comment:
I would change the name of this method to describe more closely what it does. Like, `get_processor_and_client()`